### PR TITLE
Remove googletest dependency

### PR DESCRIPTION
**Pull request by gck-ableton**
@mak-ableton @mli-ableton Any opinion?

### CODE REVIEW

**Comment by mli-ableton** [comment]
I like it!

File: src/test/CMakeLists.txt [hunk 1]
```diff
@@ -18,27 +18,20 @@
...
 add_executable(StyleSheetParserTest
+  main.cpp
   tst_Convert.cpp
   tst_CssParser.cpp
   tst_StyleMatchTree.cpp
   tst_UrlUtils.cpp
 )
 
+target_include_directories(StyleSheetParserTest PUBLIC
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include;${PROJECT_SOURCE_DIR}/third-party;${Boost_INCLUDE_DIRS}>")
```
Comment (ala-ableton) [inline src/test/CMakeLists.txt]: @gck-ableton: I don't see why you need `$<BUILD_INTERFACE:`.

File: src/test/tst_Convert.cpp [hunk 2]
```diff
@@ -25,378 +25,361 @@ THE SOFTWARE.
 #include "Warnings.hpp"
 
 SUPPRESS_WARNINGS
+#include "catch/catch.hpp"
```
Comment (ala-ableton) [inline src/test/tst_Convert.cpp]: @gck-ableton: please use angular brackets for consistency

File: src/test/tst_CssParser.cpp [hunk 3]
```diff
@@ -25,9 +25,9 @@ THE SOFTWARE.
 #include "Warnings.hpp"
 
 SUPPRESS_WARNINGS
+#include "catch/catch.hpp"
```
Comment (ala-ableton) [inline src/test/tst_CssParser.cpp]: @gck-ableton: please use angular brackets for consistency

File: src/test/tst_StyleMatchTree.cpp [hunk 4]
```diff
@@ -27,13 +27,12 @@ THE SOFTWARE.
 #include "Warnings.hpp"
 
 SUPPRESS_WARNINGS
+#include "catch/catch.hpp"
```
Comment (ala-ableton) [inline src/test/tst_StyleMatchTree.cpp]: @gck-ableton: please use angular brackets for consistency

File: src/test/tst_UrlUtils.cpp [hunk 5]
```diff
@@ -26,12 +26,12 @@ THE SOFTWARE.
 #include "Warnings.hpp"
 
 SUPPRESS_WARNINGS
+#include "catch/catch.hpp"
```
Comment (ala-ableton) [inline src/test/tst_UrlUtils.cpp]: @gck-ableton: please use angular brackets for consistency

File: src/test/CMakeLists.txt (hunk 1 above)
Comment (gck-ableton) [inline src/test/CMakeLists.txt]: I don't need it.  It's a copy-paste thing.

**Pull request merged**